src/components/panel/panel.js

```diff
@@ -1737,14 +1741,36 @@ MdPanelRef.prototype._configureClickOutsideToClose = function() {
     // panel we don't want to panel to close.
     var self = this;
     var mouseupHandler = function(ev) {
-      if (sourceElem === target[0] && ev.target === target[0]) {
+      if (self.config['propagateContainerEvents']) {
+
+        // We check if the sourceElem of the event is the panel element or one
+        // of it's children. If it is not, then close the panel.
+        var found = findSourceElem(
```

Comment:
El instead of Elem. We use el throughout this file, so it's strange to switch naming conventions here.